src/static/css/2019.css

```diff
@@ -1449,3 +1445,103 @@ p.copyright a {
     transition-delay: 0s !important;
   }
 }
+
+/* Syntax Highlighting */
+pre {
+  margin: 0;
+  border: 1px solid #ccc;
```

Comment:
Let's remove this border to minimize change (and also doesn't look great on Methodology page):

```suggestion
```